doc/container.md

```diff
@@ -83,26 +87,22 @@ The former is useful mostly for testing, or running single commands in
 an image.  It is a wrapper for `podman run -it --rm ...`, while the
 latter is a wrapper and adaptation of `podman create ...`.
 
-The second create a container with a semi-persistent writable layer that
-survives container restarts and host system restarts.  However, if you
-change the container configuration or upgrade the image (see below), the
-container will be recreated and the writable layer is lost.  This is why
-it is recommended to set up a named volume for directories, or use file
-[Content Mounts](#content-mounts), in your container if you want truly
-persistent content.
+The second creates a read-only container that is automatically started
+at every boot.  It can be extended with writable volumes that survive
+both container and host system restarts, and upgrades.  Another option
+is [Content Mounts](#content-mounts), for truly persistent content.
```

Comment:
"survive" tripped my swinglish-detector. Maybe something like:

> at every boot. When non-volatile storage is needed, data stored in [volumes](#mounts-and-volumes) will be persisted until explicitly removed from the configuration, i.e., across host and container reboots and upgrades.

Re: content mounts: are they really "truly persistent"?

Reply:
Did this:

> The second creates a read-only container that is automatically started
> at every boot.  When non-volatile storage is needed, data stored in a
> volume is persisted until explicitly removed from the configuration,
> i.e., across host and container reboots and upgrades.
> 
> Another option is [Content Mounts](#content-mounts), where the content
> of a file mounted into the container is kept along with the container
> configuration in the device's `startup-config`.